aim/agent/aid/universes/aci/aci_universe.py

```diff
@@ -263,10 +413,10 @@ def _thread_monitor(self, flag):
             utils.perform_harakiri(LOG, msg)
 
 
-def get_websocket_context(apic_config):
+def get_websocket_context(apic_config, aim_manager):
     global ws_context
     if not ws_context:
-        ws_context = WebSocketContext(apic_config)
+        ws_context = WebSocketContext(apic_config, aim_manager)
```

Comment:
I can certainly add a REVISIT. Or if u prefer, for now I can initialize a new aim_manager member variable inside WebSocketContext. I passed it in in this patch because aci_universe already has an aim_manager member variable available so.

Reply:
I think the REVISIT is good for now - passing explicitly is probably best here.